internal/bind/bind_allocator.go

```diff
@@ -35,7 +35,11 @@ func (b *BindAllocator) Next() *url.URL {
 	var err error
 	for {
 		b.port.Add(1)
-		l, err = net.ListenTCP("tcp", &net.TCPAddr{IP: net.ParseIP(b.baseURL.Hostname()), Port: int(b.port.Load())})
+		host := b.baseURL.Hostname()
```

Comment:
Let's just change the default addresses to `127.0.0.1` then we don't need to special case this here.